t/cobrand/brent.t

```diff
@@ -102,4 +102,26 @@ subtest "UnitID on gully sent across in detail" => sub {
     };
 };
 
+subtest "Report title passed through as attribute" => sub {
+    my ($problem) = $mech->create_problems_for_body(1, $brent->id, 'Gully', {
+        areas => "2488", category => 'Gully grid missing', cobrand => 'brent',
+    });
+
+    FixMyStreet::override_config {
+        ALLOWED_COBRANDS => 'brent',
+        MAPIT_URL => 'http://mapit.uk/',
+        STAGING_FLAGS => { send_reports => 1 },
+        COBRAND_FEATURES => {
+            anonymous_account => {
+                brent => 'anonymous'
+            },
+        },
+    }, sub {
+        FixMyStreet::Script::Reports::send();
+        my $req = Open311->test_req_used;
+        my $c = CGI::Simple->new($req->content);
+        is $c->param('attribute[title]'), 'Gully Test 1 for 1', 'Title passed as attribute for Open311';
```

Comment:
Ah you have used a fixed ID here, that won't work, you'll need to make sure it's the actual $problem->id